docs/bucket/replication/README.md

```diff
@@ -145,6 +147,11 @@ Note that both source and target instance need to be upgraded to latest release
 
 ![delete](https://raw.githubusercontent.com/minio/minio/master/docs/bucket/replication/DELETE_bucket_replication.png)
 
+The status of replication can be monitored by configuring event notifications on the source and target buckets using `mc event add`.On the source side, the `s3:PutObject`, `s3:Replication:OperationCompletedReplication` and `s3:Replication:OperationFailedReplication` events show the status of replication in the `X-Amz-Replication-Status` metadata.
+
+On the target bucket, `s3:PutObject` event shows `X-Amz-Replication-Status` status of `REPLICA` in the metadata. Additional metrics to monitor backlog state for the purpose of bandwidth management and resource allocation are  
+an upcoming feature.
+
```

Comment:
You can talk about x-minio-replication-delete-status?

Reply:
done, also moved the commit from other PR to show `x-minio-replication-delete-status` as it is more relevant here.